src/components/ProgressBar/__tests__/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+
+import { NAMESPACE } from '../../../constants'
+import { ProgressBar } from '../'
+
+describe('<ProgressBar />', () => {
+  it('renders a default className', () => {
+    const $ = shallow(<ProgressBar>_</ProgressBar>)
+    expect($.hasClass(`${NAMESPACE}c-progress-bar`)).toBe(true)
+  })
+
+  it('passes in an optional className override', () => {
+    const $ = shallow(<ProgressBar className="test">_</ProgressBar>)
+    expect($.hasClass('test')).toBe(true)
+  })
+
+  it('passes in a "percent" prop and renders an inline style width as a percentage', () => {
+    const $ = mount(<ProgressBar percent="50" style={{ width: '50%' }}>_</ProgressBar>)
+    expect($.prop('percent')).toBe('50')
```

Comment:
This test is essentially testing your test and not the component, the test below is all we need to assert the behaviour we are looking for 😄

Reply:
done